test/test-ghe-backup.sh

```diff
@@ -214,6 +213,76 @@ begin_test "ghe-backup subsequent snapshot"
 )
 end_test
 
+begin_test "ghe-backup with relative data dir path"
+(
+    set -e
+
+    # wait a second for snapshot timestamp
+    sleep 1
+
+    # grab the previous snapshot number so we can compare after
+    previous_snapshot=$(ls -ld "$GHE_DATA_DIR/current" | sed 's/.* -> //')
+
+    # Change working directory to the root directory
+    cd $ROOTDIR
+
+    # run it
+    GHE_DATA_DIR=$(echo $GHE_DATA_DIR | sed 's|'$ROOTDIR'/||') ghe-backup
+
+    # check that current symlink points to new snapshot
+    this_snapshot=$(ls -ld "$GHE_DATA_DIR/current" | sed 's/.* -> //')
+    [ "$previous_snapshot" != "$this_snapshot" ]
+
```

Comment:
Nit but rather than setting and comparing `previous_snapshot` and `this_snapshot`, I'd set `GHE_SNAPSHOT_TIMESTAMP`, making sure it's there after the backup. That way we don't need all the sed magic with the symlinks:

```
cd $ROOTDIR
export GHE_SNAPSHOT_TIMESTAMP=123-relative
export GHE_DATA_DIR=$(echo $GHE_DATA_DIR | sed 's|'$ROOTDIR'/||')
ghe-backup
```